lightyear/src/connection/netcode/server.rs

```diff
@@ -474,7 +474,7 @@ impl<Ctx> NetcodeServer<Ctx> {
         &mut self,
         addr: SocketAddr,
         packet: Packet,
-        sender: &mut impl PacketSender,
+        sender: &mut dyn PacketSender,
```

Comment:
Why `dyn` instead of `impl`? Is it to keep the types simpler?

Reply:
Not too sure, I think the compiler was shouting at me :)